File: python/triangulo.py
from figuras2d import Figuras2D

class Triangulo(Figuras2D):
    def __init__(self, base, altura):
        self.base = base
        self.altura = altura
        
    def calcularArea(self):
        area = (self.base * self.altura)/2
        return area
        

File: Java/Triangulo.java
public class Triangulo extends Figuras2d{
    protected  double altura;
    protected double base;

    public Triangulo(String nombre, double altura, double base){
        super(nombre);
        this.altura=altura;
        this.base=base;
    }

    @Override
    public double calcularArea(){
        //formula areaTriangulo: (base*altura)/2;
        return (base*altura)/2;
        
    }
    
}
